include/deal.II/numerics/vector_tools.h

```diff
@@ -763,22 +763,26 @@ namespace VectorTools
    * values are disabled. The ordering of arguments to this function is such
    * that you need not give a second quadrature formula if you don't want to
    * project to the boundary first, but that you must if you want to do so.
+
+   * If the FiniteElement is supported by the MatrixFree class, neither
```

Comment:
Or maybe write: "A matrix-free implementation is used when: (i)...(ii)... (iii).... Otherwise..."